### ISSUE TITLE
On the difficulty of training Recurrent Neural Networks

### ISSUE THREAD

**Issue by TanUkkii007** [TanUkkii007/papers-i-read#98]
Former title: Understanding the exploding gradient problem

link: https://arxiv.org/abs/1211.5063

referenced from:
- Zoneout: Regularizing RNNs by randomly preserving hidden activations (for clip gradients to a maximum norm of 1) #75
